server/src/sim/test_makeworkbook.py

```diff
@@ -46,8 +46,12 @@ def test_ref_years_range(self):
 class TestOptimaGraphTable(unittest.TestCase):
 
     def test_create_table(self):
-        table = OptimaGraphTable('Beautiful graph', [{'title':'one', 'data':[1,2,3]}, \
-            {'title':'two', 'data':[2,3,5]},{'title':'three', 'data':["a","b","c"]}])
+        sheet = [{
+            "name":"GRAPH DATA",
+            "columns":[{'title':'one', 'data':[1,2,3]}, \
+            {'title':'two', 'data':[2,3,5]},{'title':'three', 'data':["a","b","c"]}]
+        }]
+        table = OptimaGraphTable('Beautiful graph', sheet)
         path = '/tmp/test_graph.xlsx'
```

Comment:
I find this test wrong in several ways: 1. given a sheet name I would expect to be able to find an excel file with that name. 2. the design of OptimaGraphTable forces me to invoke create that should not be necessary 3. we are only testing the existence of a file without checking the actual content